[2m2025-07-05T11:13:39Z[0m runner[7840076ce9e978] [32mfra[0m [[34minfo[0m]Machine started in 1.306s
[2m2025-07-05T11:13:40Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m]2025/07/05 11:13:40 INFO SSH listening listen_address=[fdaa:20:27fd:a7b:3fa:392b:b2dc:2]:22
[2m2025-07-05T11:13:40Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m]Traceback (most recent call last):
[2m2025-07-05T11:13:40Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m]  File "/app/main.py", line 2, in <module>
[2m2025-07-05T11:13:40Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m]    from router import route_request
[2m2025-07-05T11:13:40Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m]  File "/app/router.py", line 36
[2m2025-07-05T11:13:40Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m]    if 'I can∩┐╜t' in response or 'I∩┐╜m afraid' in response or 'I won∩┐╜t' in response:
[2m2025-07-05T11:13:40Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m]               ^^
[2m2025-07-05T11:13:40Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m]SyntaxError: (unicode error) 'utf-8' codec can't decode byte 0x92 in position 5: invalid start byte
[2m2025-07-05T11:13:40Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m][32m INFO[0m Main child exited normally with code: 1
[2m2025-07-05T11:13:40Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m][32m INFO[0m Starting clean up.
[2m2025-07-05T11:13:40Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m][33m WARN[0m could not unmount /rootfs: EINVAL: Invalid argument
[2m2025-07-05T11:13:40Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m][    1.901578] reboot: Restarting system
[2m2025-07-05T11:13:43Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m]2025-07-05T11:13:43.898146722 [01JZD4MTNS642KVYN40WX96WFM:main] Running Firecracker v1.7.0
[2m2025-07-05T11:13:44Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m][32m INFO[0m Starting init (commit: d0572327e)...
[2m2025-07-05T11:13:44Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m][32m INFO[0m Preparing to run: `python main.py` as root
[2m2025-07-05T11:13:44Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m][32m INFO[0m [fly api proxy] listening at /.fly/api
[2m2025-07-05T11:13:45Z[0m runner[7840076ce9e978] [32mfra[0m [[34minfo[0m]Machine started in 1.427s
[2m2025-07-05T11:13:45Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m]2025/07/05 11:13:45 INFO SSH listening listen_address=[fdaa:20:27fd:a7b:3fa:392b:b2dc:2]:22
[2m2025-07-05T11:13:45Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m]Traceback (most recent call last):
[2m2025-07-05T11:13:45Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m]  File "/app/main.py", line 2, in <module>
[2m2025-07-05T11:13:45Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m]    from router import route_request
[2m2025-07-05T11:13:45Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m]  File "/app/router.py", line 36
[2m2025-07-05T11:13:45Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m]    if 'I can∩┐╜t' in response or 'I∩┐╜m afraid' in response or 'I won∩┐╜t' in response:
[2m2025-07-05T11:13:45Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m]               ^^
[2m2025-07-05T11:13:45Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m]SyntaxError: (unicode error) 'utf-8' codec can't decode byte 0x92 in position 5: invalid start byte
[2m2025-07-05T11:13:45Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m][32m INFO[0m Main child exited normally with code: 1
[2m2025-07-05T11:13:45Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m][32m INFO[0m Starting clean up.
[2m2025-07-05T11:13:45Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m][33m WARN[0m could not unmount /rootfs: EINVAL: Invalid argument
[2m2025-07-05T11:13:45Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m][    1.939211] reboot: Restarting system
[2m2025-07-05T11:13:47Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m]2025-07-05T11:13:47.106015765 [01JZD4MTNS642KVYN40WX96WFM:main] Running Firecracker v1.7.0
[2m2025-07-05T11:13:47Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m][32m INFO[0m Starting init (commit: d0572327e)...
[2m2025-07-05T11:13:48Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m][32m INFO[0m Preparing to run: `python main.py` as root
[2m2025-07-05T11:13:48Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m][32m INFO[0m [fly api proxy] listening at /.fly/api
[2m2025-07-05T11:13:48Z[0m runner[7840076ce9e978] [32mfra[0m [[34minfo[0m]Machine started in 1.383s
[2m2025-07-05T11:13:48Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m]2025/07/05 11:13:48 INFO SSH listening listen_address=[fdaa:20:27fd:a7b:3fa:392b:b2dc:2]:22
[2m2025-07-05T11:13:48Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m]Traceback (most recent call last):
[2m2025-07-05T11:13:48Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m]  File "/app/main.py", line 2, in <module>
[2m2025-07-05T11:13:48Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m]    from router import route_request
[2m2025-07-05T11:13:48Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m]  File "/app/router.py", line 36
[2m2025-07-05T11:13:48Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m]    if 'I can∩┐╜t' in response or 'I∩┐╜m afraid' in response or 'I won∩┐╜t' in response:
[2m2025-07-05T11:13:48Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m]               ^^
[2m2025-07-05T11:13:48Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m]SyntaxError: (unicode error) 'utf-8' codec can't decode byte 0x92 in position 5: invalid start byte
[2m2025-07-05T11:13:49Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m][32m INFO[0m Main child exited normally with code: 1
[2m2025-07-05T11:13:49Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m][32m INFO[0m Starting clean up.
[2m2025-07-05T11:13:49Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m][33m WARN[0m could not unmount /rootfs: EINVAL: Invalid argument
[2m2025-07-05T11:13:49Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m][    1.901931] reboot: Restarting system
[2m2025-07-05T11:13:59Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m]2025-07-05T11:13:59.049803063 [01JZD4MTNS642KVYN40WX96WFM:main] Running Firecracker v1.7.0
[2m2025-07-05T11:13:59Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m][32m INFO[0m Starting init (commit: d0572327e)...
[2m2025-07-05T11:14:00Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m][32m INFO[0m Preparing to run: `python main.py` as root
[2m2025-07-05T11:14:00Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m][32m INFO[0m [fly api proxy] listening at /.fly/api
[2m2025-07-05T11:14:00Z[0m runner[7840076ce9e978] [32mfra[0m [[34minfo[0m]Machine started in 1.493s
[2m2025-07-05T11:14:00Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m]2025/07/05 11:14:00 INFO SSH listening listen_address=[fdaa:20:27fd:a7b:3fa:392b:b2dc:2]:22
[2m2025-07-05T11:14:00Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m]Traceback (most recent call last):
[2m2025-07-05T11:14:00Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m]  File "/app/main.py", line 2, in <module>
[2m2025-07-05T11:14:00Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m]    from router import route_request
[2m2025-07-05T11:14:00Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m]  File "/app/router.py", line 36
[2m2025-07-05T11:14:00Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m]    if 'I can∩┐╜t' in response or 'I∩┐╜m afraid' in response or 'I won∩┐╜t' in response:
[2m2025-07-05T11:14:00Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m]               ^^
[2m2025-07-05T11:14:00Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m]SyntaxError: (unicode error) 'utf-8' codec can't decode byte 0x92 in position 5: invalid start byte
[2m2025-07-05T11:14:01Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m][32m INFO[0m Main child exited normally with code: 1
[2m2025-07-05T11:14:01Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m][32m INFO[0m Starting clean up.
[2m2025-07-05T11:14:01Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m][33m WARN[0m could not unmount /rootfs: EINVAL: Invalid argument
[2m2025-07-05T11:14:01Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m][    1.907418] reboot: Restarting system
[2m2025-07-05T11:14:23Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m]2025-07-05T11:14:23.941067676 [01JZD4MTNS642KVYN40WX96WFM:main] Running Firecracker v1.7.0
[2m2025-07-05T11:14:24Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m][32m INFO[0m Starting init (commit: d0572327e)...
[2m2025-07-05T11:14:24Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m][32m INFO[0m Preparing to run: `python main.py` as root
[2m2025-07-05T11:14:24Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m][32m INFO[0m [fly api proxy] listening at /.fly/api
[2m2025-07-05T11:14:24Z[0m runner[7840076ce9e978] [32mfra[0m [[34minfo[0m]Machine started in 1.638s
[2m2025-07-05T11:14:25Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m]2025/07/05 11:14:25 INFO SSH listening listen_address=[fdaa:20:27fd:a7b:3fa:392b:b2dc:2]:22
[2m2025-07-05T11:14:25Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m]Traceback (most recent call last):
[2m2025-07-05T11:14:25Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m]  File "/app/main.py", line 2, in <module>
[2m2025-07-05T11:14:25Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m]    from router import route_request
[2m2025-07-05T11:14:25Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m]  File "/app/router.py", line 36
[2m2025-07-05T11:14:25Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m]    if 'I can∩┐╜t' in response or 'I∩┐╜m afraid' in response or 'I won∩┐╜t' in response:
[2m2025-07-05T11:14:25Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m]               ^^
[2m2025-07-05T11:14:25Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m]SyntaxError: (unicode error) 'utf-8' codec can't decode byte 0x92 in position 5: invalid start byte
[2m2025-07-05T11:14:25Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m][32m INFO[0m Main child exited normally with code: 1
[2m2025-07-05T11:14:25Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m][32m INFO[0m Starting clean up.
[2m2025-07-05T11:14:25Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m][33m WARN[0m could not unmount /rootfs: EINVAL: Invalid argument
[2m2025-07-05T11:14:25Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m][    1.928744] reboot: Restarting system
[2m2025-07-05T11:14:42Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m]2025-07-05T11:14:42.776090894 [01JZD4MTNS642KVYN40WX96WFM:main] Running Firecracker v1.7.0
[2m2025-07-05T11:14:43Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m][32m INFO[0m Starting init (commit: d0572327e)...
[2m2025-07-05T11:14:43Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m][32m INFO[0m Preparing to run: `python main.py` as root
[2m2025-07-05T11:14:43Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m][32m INFO[0m [fly api proxy] listening at /.fly/api
[2m2025-07-05T11:14:43Z[0m runner[7840076ce9e978] [32mfra[0m [[34minfo[0m]Machine started in 1.472s
[2m2025-07-05T11:14:44Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m]2025/07/05 11:14:44 INFO SSH listening listen_address=[fdaa:20:27fd:a7b:3fa:392b:b2dc:2]:22
[2m2025-07-05T11:14:44Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m]Traceback (most recent call last):
[2m2025-07-05T11:14:44Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m]  File "/app/main.py", line 2, in <module>
[2m2025-07-05T11:14:44Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m]    from router import route_request
[2m2025-07-05T11:14:44Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m]  File "/app/router.py", line 36
[2m2025-07-05T11:14:44Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m]    if 'I can∩┐╜t' in response or 'I∩┐╜m afraid' in response or 'I won∩┐╜t' in response:
[2m2025-07-05T11:14:44Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m]               ^^
[2m2025-07-05T11:14:44Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m]SyntaxError: (unicode error) 'utf-8' codec can't decode byte 0x92 in position 5: invalid start byte
[2m2025-07-05T11:14:44Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m][32m INFO[0m Main child exited normally with code: 1
[2m2025-07-05T11:14:44Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m][32m INFO[0m Starting clean up.
[2m2025-07-05T11:14:44Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m][33m WARN[0m could not unmount /rootfs: EINVAL: Invalid argument
[2m2025-07-05T11:14:44Z[0m app[7840076ce9e978] [32mfra[0m [[34minfo[0m][    1.895033] reboot: Restarting system
[2m2025-07-05T11:14:45Z[0m runner[7840076ce9e978] [32mfra[0m [[34minfo[0m]machine has reached its max restart count of 10
[2m2025-07-05T11:17:42Z[0m runner[7840076ce9e978] [32mfra[0m [[34minfo[0m]Pulling container image registry.fly.io/lucien-outside@sha256:67c27c228369c76c6983fc1064ea598ed5babe78d6e2744a35ee8a8690dde41c
